src/converter/one/convert/JfrToOtlp.java

```diff
@@ -28,6 +28,8 @@ public class JfrToOtlp extends JfrConverter {
     private final Index<String> functionPool = new Index<>(String.class, "");
     private final Index<Line> linePool = new Index<>(Line.class, Line.EMPTY);
     private final Index<KeyValue> attributesPool = new Index<>(KeyValue.class, KeyValue.EMPTY);
+    private final Index<int[]> stacksPool = new Index<>(int[].class, new int[0]);
```

Comment:
Index, being a HashMap, does not work for arrays. You'll need a wrapper that implements equals/hashCode.

Reply:
TIL, had no idea. Why is it done like that?

Reply:
Yeah, I meant why doesn't array have a reasonable implementation for `equals` and `hashCode`. I'll go for a wrapper, thanks.